map/bookmark_helpers.cpp

```diff
@@ -160,6 +160,10 @@ std::map<std::string, BookmarkMatchInfo> const kFeatureTypeToBookmarkMatchInfo =
   {"tourism-attraction", {kml::BookmarkIcon::Sights, BookmarkBaseType::Sights}},
   {"waterway-waterfall", {kml::BookmarkIcon::Sights, BookmarkBaseType::Sights}},
 
+  {"tourism-information", {kml::BookmarkIcon::Information, BookmarkBaseType::Sights}},
+  {"tourism-information-office", {kml::BookmarkIcon::Information, BookmarkBaseType::Sights}},
+  {"tourism-information-visitor_centre", {kml::BookmarkIcon::Information, BookmarkBaseType::Sights}},
```

Comment:
ToDo: Check here and around if it is really necessary to add parent type and its children.